markerclustererplus/src/markerclusterer.js

```diff
@@ -286,6 +298,10 @@ ClusterIcon.prototype.show = function () {
       this.div_.title = this.sums_.title;
```

Comment:
Just before this line you should also be setting the innerHTML of the overlayMouseTarget div to the same as that of the overlayLayer div. This ensures the overlayMouseTarget div has exactly the same size as the overlayLayer div.

Reply:
Why wouldn't copying the height and width from the overlayDiv work without duplicating innerHTML?

I'm open to setting it and adding `opacity: 0` to line 388; it just seems unnecessary.

Reply:
I'm just being cautious.